recipes/continuable/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class ContinuableConan(ConanFile):
+    name = "continuable"
+    description = "C++14 asynchronous allocation aware futures (supporting then, exception handling, coroutines and connections)"
+    topics = "asynchronous", "future", "coroutines", "header-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Naios/continuable"
+    license = "MIT"
+    settings = "os", "compiler"
+    no_copy_source = True
+    requires = (
+        "function2/4.1.0",
+    )
+    options = {
+        # Exceptions are disabled and `std::error_condition` is used as error_type. See tutorial-chaining-continuables-fail for details.
+        "no_exceptions": [True, False],
+        # Exceptions are disabled and the type defined by `CONTINUABLE_WITH_CUSTOM_ERROR_TYPE` is used as error_type.
+        # See tutorial-chaining-continuables-fail for details.
+        "custom_error_type": [True, False],
+        # Allows unhandled exceptions in asynchronous call hierarchies. See tutorial-chaining-continuables-fail for details.
+        "unhandled_exceptions": [True, False],
+        # Allows to customize the final callback which can be used to implement custom unhandled asynchronous exception handlers.
+        "custom_final_callback": [True, False],
+        # Don"t decorate the used type erasure, which is done to keep type names minimal for better error messages in debug builds.
+        "immediate_types": [True, False],
+    }
+    default_options = {
+        "no_exceptions": False,
+        "custom_error_type": False,
+        "unhandled_exceptions": False,
+        "custom_final_callback": False,
+        "immediate_types": False
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
```

Comment:
```suggestion
    def validate(self):
```

Reply:
Oh since when does this exist and what is the difference?

Reply:
1.28.0 as I recall